auth_interfaces.go

```diff
@@ -58,3 +59,9 @@ type UserManager interface {
 	// Returns the groups or roles to which a user belongs
 	GetGroupsForUser(string) ([]string, error)
 }
+
+type RoleManager interface {
```

Comment:
docstring for the type but also for each method as a hint to implenters.

Reply:
added